### PR TITLE
[8.0] l10n_it_fatturapa: import guards

### PR DESCRIPTION
**Pull request by LeartS**

### CODE REVIEW

File: l10n_it_fatturapa/bindings/__init__.py [hunk 1]
```diff
@@ -17,5 +17,7 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-
-from . import fatturapa_v_1_1
+try:
+    from . import fatturapa_v_1_1
+except ImportError:
+    pass # Module is not installed, avoid init error
```
Comment (eLBati) [inline l10n_it_fatturapa/bindings/__init__.py]: Why this? When you use this module, file fatturapa_v_1_1.py is always present

Comment (eLBati) [inline l10n_it_fatturapa/bindings/__init__.py]: Oh I see: pyxb is imported several times in file fatturapa_v_1_1.py
Could you just add a comment explaining it?

**Comment by eLBati** [comment]
Thanks @LeartS please also see travis: ```l10n_it_fatturapa/bindings/__init__.py:23:9: E261 at least two spaces before inline comment```

File: l10n_it_fatturapa/bindings/__init__.py (hunk 1 above)
Comment (LeartS) [inline l10n_it_fatturapa/bindings/__init__.py]: Yep that's it, there are a lot of statements (not only imports) that need
to be guarded against. You are right, a comment would be nice, I'll add
one.

File: l10n_it_fatturapa_out/wizard/__init__.py [hunk 2]
```diff
@@ -18,4 +18,7 @@
 #
 ##############################################################################
 
-from . import wizard_export_fatturapa
+try:
```
Comment (archetipo) [inline l10n_it_fatturapa_out/wizard/__init__.py]: Why this?

Comment (eLBati) [inline l10n_it_fatturapa_out/wizard/__init__.py]: See https://github.com/OCA/l10n-italy/pull/164/files#r46173549

Comment (archetipo) [inline l10n_it_fatturapa_out/wizard/__init__.py]: ok but is useless check here, if the  l10n_it_fatturapa module is not resolved, or not?

File: l10n_it_fatturapa/bindings/__init__.py (hunk 1 above)
Comment (archetipo) [inline l10n_it_fatturapa/bindings/__init__.py]: Please add a log as written in https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror

**Comment by LeartS** [comment]
@eLBati @archetipo I addressed your comments

**Comment by eLBati** [comment]
thanks :+1:

**Comment by archetipo** [comment]
@LeartS  thanks  :+1:

**Pull request merged**